packages/@aws-cdk/aws-ec2/lib/bastion-host.ts

```diff
@@ -0,0 +1,73 @@
+import { PolicyStatement } from "@aws-cdk/aws-iam";
+import { CfnOutput, Construct } from "@aws-cdk/core";
+import { AmazonLinuxGeneration, AmazonLinuxImage, InstanceClass, InstanceSize, InstanceType } from ".";
+import { Instance } from "./instance";
+import { ISecurityGroup } from "./security-group";
+import { IVpc, SubnetType } from "./vpc";
+
+/**
+ * Properties of the bastion host
+ */
+export interface BastionHostLinuxProps {
+
+  /**
+   * In which AZ to place the instance within the VPC
+   *
+   * @default - Random zone.
+   */
+  readonly availabilityZone?: string;
+
+  /**
+   * VPC to launch the instance in.
+   */
+  readonly vpc: IVpc;
+
+  /**
+   * The name of the instance
+   *
+   * @default - CDK generated name
+   */
+  readonly instanceName?: string;
+
+  /**
+   * Use public subnet instead of private one
+   *
+   * @default - false
+   */
+  readonly publicSubnets?: boolean;
+
+  /**
+   * Security Group to assign to this instance
+   *
+   * @default - create new security group
+   */
+  readonly securityGroup?: ISecurityGroup;
+}
+
+export class BastionHostLinux extends Instance {
```

Comment:
Please describe what kind of bastion you're getting in more detail.

Also, please mark this class `@experimental` to start with.